ga4gh/frontend.py

```diff
@@ -40,11 +58,23 @@ def handleHTTPOptions():
     response.headers.add("Access-Control-Request-Methods", "GET,POST,OPTIONS")
     return response
 
-
 @app.route('/')
 def index():
-    flask.abort(404)
+    if app.config["BEACON"]:
+        # TODO: Get list of variant_sets from backend
+        return flask.render_template('beaconClient.html', variant_sets=app.backend._variantSetIdMap)
+    else:
+        flask.abort(404)
+
+
+if app.config["BEACON"]:
```

Comment:
This won't work in general, because the config won't be set at module load time.